### PR TITLE
Bug: Storm cli completion

### PR DESCRIPTION
**Pull request by vEpiphyte**
Support older cortex without coreinfov2 update
Ensure that tag completion runs in the current view

### CODE REVIEW

**Review by MichaelSquires — APPROVED** [review]

File: synapse/tools/storm.py [hunk 1]
```diff
@@ -349,8 +349,7 @@ async def _get_tag_completions(self, prefix='', limit=100):
         +:depth<=$depth
         | uniq
         | limit $limit
-        | $leaf = $lib.true { -> syn:tag:up | limit 1 | $leaf = $lib.false }
-        $doc = ''
+        | $doc = ''
         if $node.props.doc {
```
Comment (invisig0th) [inline synapse/tools/storm.py:353]: why did you add a pipe here?

Comment (MichaelSquires) [reply synapse/tools/storm.py:353]: The previous line was removed (the one with `$leaf = ...`) because it wasn't being used.

Comment (vEpiphyte) [reply synapse/tools/storm.py:353]: This is after a `limit` command. The `$leaf` value is unused.
